nuspec/CoreContent/MainViewModel.cs.pp

```diff
@@ -0,0 +1,45 @@
+using MvvmCross.Core.ViewModels;
+
+namespace $rootnamespace$.ViewModels
+{
+    public class MainViewModel : MvxViewModel
+    {
+        public MainViewModel()
+        {
+            ResetTextCommand = new MvxCommand(ResetTextCommandExecute);
+        }
+
+        // Even though there's no Init method on MvxViewModel, this method is used by
```

Comment:
I'm not sure if this documentation should be in here. Rather have this on the website.
I would have the Init method enabled here.